ibis/common/patterns.py

```diff
@@ -840,6 +868,8 @@ class GenericInstanceOf(Slotted, Pattern):
     """
 
     __slots__ = ("origin", "fields")
+    origin: type
+    fields: dict
```

Comment:
Theoretically this should be `FrozenDict[str, Pattern]`

Reply:
Done